src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -1,6 +1,25 @@
-﻿// ****************************************************************
-// Copyright (c) 2011-2015 NUnit Software. All rights reserved.
-// ****************************************************************
+﻿// ***********************************************************************
+// Copyright (c) 2011-2015 Charlie Poole
```

Comment:
It's something that has been disputed over the years, for sure. Most people using MIT seem to put the entire thing in each header. The shorter notice is a holdover from when this was licensed differently. I think most people writing about copyright and license say to put a notice in the files, while those merely concerned with coding issues will discount it. We can discuss more - or more likely, you guys can after I'm gone. 😄